main.go

```diff
@@ -25,19 +27,24 @@ func main() {
 	rest.SetDefaultWarningHandler(rest.NoWarnings{})
 
 	environment := "prod"
-	release := fmt.Sprintf("cli@%s", cmd.BinaryVersion)
-
 	if cmd.IsDevVersion() {
 		environment = "dev"
 	}
 
-	sentryClientOptions := sentry_utils.GetSentryClientOptions(environment, release)
+	sentryClientOptions := sentry_utils.GetSentryClientOptions(APP_NAME, environment, cmd.BinaryVersion)
 	if err = sentry.Init(sentryClientOptions); err != nil {
 		ui.GlobalWriter.PrintErrorMessageln(err.Error())
 		panic(err)
 	}
 	defer sentry.Flush(sentry_utils.FLUSH_TIMEOUT)
 
+	segmentConfig := segment.GetConfig(APP_NAME, cmd.BinaryVersion)
+	if err = segment.Init(segmentConfig); err != nil {
```

Comment:
Segment shouldn't fail the process

Reply:
The function will return an error if the configuration contained impossible, so needs to panic here